.github/workflows/riscv-no-std.yml

```diff
@@ -0,0 +1,50 @@
+name: RISC-V & no_std check
+
+concurrency:
+  cancel-in-progress: true
+  group: ${{github.workflow}}-${{github.ref}}
+
+on:
+  push:
+    branches: [main, "release/**"]
+  pull_request:
+    branches: [main, "release/**"]
+  merge_group:
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  check-no-std:
+    name: check no_std RISCV32
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    strategy:
+      fail-fast: false
+      matrix:
+        features: [""]
```

Comment:
This is here for the case where we want to possibly enable a dependency/alternative feature to see if it also compiles. Took this from revm since they have cases like this

Reply:
Removed this file due to: https://github.com/paradigmxyz/reth/pull/15786#discussion_r2049053168